java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
Now I get what you meant by “the variables in the string”. On the other hand I’m moving away from having the Bundles include the semicolon (or - not as pretty - equal sign) so we can reuse the translation in more places.
Obviously there are applications where different languages want it in a different order, for instance in German, but that applies more to verbs than interpunction.
I also stripped out localizing X, Y and the initial values for them. That would make support too hard.

> Op 19 aug. 2016, om 19:43 heeft Randall Wood <notifications@github.com> het volgende geschreven:
> I understand Level was added, but this should be: textX.setText(Bundle.getMessage("Level", pl.getDisplayLevel())); and the value in the .properties file for "Level" should be something like "level = {0}"